pom.xml

```diff
@@ -743,6 +743,7 @@
                                         <sortModulesPath>integration-tests/pom.xml</sortModulesPath>
                                         <sortModulesPath>integration-test-groups/pom.xml</sortModulesPath>
                                         <sortModulesPath>integration-test-groups/aws2/pom.xml</sortModulesPath>
+                                        <sortModulesPath>integration-test-groups/aws2-quarkus-client/pom.xml</sortModulesPath>
```

Comment:
Might be best to remove these changes as I opened a PR to specifically fix up the missing stuff https://github.com/apache/camel-quarkus/pull/3136.

Reply:
OK, I will do.